### PR TITLE
Update saas configuration URL and yq command

### PR DESCRIPTION
**Pull request by jharrington22**
This PR updates the endpoint in which we retrieve the saas configuration and also fixes the yq syntax.

### CODE REVIEW

**Comment by jharrington22** [comment]
@cblecker @lisa @jewzaam PTAL

**Comment by cblecker** [comment]
Is this going to get errored/blown away by the boilerplate?

File: functions.mk [hunk 1]
```diff
@@ -18,7 +18,7 @@ define create_push_catalog_image
 	mkdir -p bundles-$(1)/$(OPERATOR_NAME) ;\
 	removed_versions="" ;\
 	if [[ "$$(echo $(4) | tr [:upper:] [:lower:])" == "true" ]]; then \
-		deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/$(5)/raw/master/$(6)' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq -r '.services[]|select(.name="configure-alertmanager-operator").hash') ;\
+	deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/service/app-interface/-/raw/master/data/services/osd-operators/cicd/saas/saas-configure-alertmanager-operator.yaml' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq yq r - 'resourceTemplates[*].targets(namespace.$ref===/services/osd-operators/namespaces/hivep01ue1/cluster-scope.yml).ref') ;\
```
Comment (lisa) [inline functions.mk:21]: Whitespace changes here, and raise awareness of the abandonment of input variables to this function. The former is more important than the latter, in my opinion.

**Review by lisa — CHANGES_REQUESTED** [review]
/hold

File: functions.mk (hunk 1 above)
Comment (cblecker) [inline functions.mk:21]:
```suggestion
	deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/service/app-interface/-/raw/master/data/services/osd-operators/cicd/saas/saas-configure-alertmanager-operator.yaml' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq yq r - 'resourceTemplates[*].targets(namespace.$ref==/services/osd-operators/namespaces/hivep01ue1/cluster-scope.yml).ref') ;\
```

**Comment by 2uasimojo** [comment]
> Is this going to get errored/blown away by the boilerplate?

I sure hope so.

File: functions.mk [hunk 2]
```diff
@@ -18,7 +18,7 @@ define create_push_catalog_image
 	mkdir -p bundles-$(1)/$(OPERATOR_NAME) ;\
 	removed_versions="" ;\
 	if [[ "$$(echo $(4) | tr [:upper:] [:lower:])" == "true" ]]; then \
-		deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/$(5)/raw/master/$(6)' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq -r '.services[]|select(.name="configure-alertmanager-operator").hash') ;\
+	deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/service/app-interface/-/raw/master/data/services/osd-operators/cicd/saas/saas-configure-alertmanager-operator.yaml' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq yq r - 'resourceTemplates[*].targets(namespace.$ref==/services/osd-operators/namespaces/hivep01ue1/cluster-scope.yml).ref') \;
```
Comment (cblecker) [inline functions.mk:21]:
```suggestion
		deployed_hash=$$(curl -s 'https://gitlab.cee.redhat.com/service/app-interface/-/raw/master/data/services/osd-operators/cicd/saas/saas-configure-alertmanager-operator.yaml' | $(CONTAINER_ENGINE) run --rm -i quay.io/app-sre/yq yq r - 'resourceTemplates[*].targets(namespace.$ref==/services/osd-operators/namespaces/hivep01ue1/cluster-scope.yml).ref') ;\
```

**Comment by 2uasimojo** [comment]
> > Is this going to get errored/blown away by the boilerplate?
> 
> I sure hope so.

Sorry, I misunderstood the question.

The current versions of boilerplate (the one used by this repo, and the latest in bp itself) don't touch anything under the umbrella of hack/app_sre_build_deploy.sh, so this will be safe until we start ripping that apart.

And when we do, whatever we figure out to replace this will be better in at least the following ways:
- It will not have a 2.5KB one-line bash script inlined in a `make` function.
- It will not hardcode hive env names when trying to discover the previously-deployed CSV. (Though I have no idea yet how we're actually going to fix this.)

**Pull request merged**